sdv/sequential/par.py

```diff
@@ -73,6 +73,9 @@ def _get_context_metadata(self):
 
         for column in context_columns:
             context_columns_dict[column] = self.metadata.columns[column]
+            # Context datetime SDTypes for PAR have already been converted to float timestamp
```

Comment:
Should we add a check here to confirm that the datetime has actually been converted to float first before we change the sdtype?

Reply:
Added!